pallets/connectors/src/lib.rs

```diff
@@ -434,7 +500,8 @@ pub mod pallet {
 				OriginKind::SovereignAccount,
 				TransactWeights {
 					// Specify a conservative max weight
-					transact_required_weight_at_most: 8_000_000_000,
+					transact_required_weight_at_most: xcm_domain.max_gas_limit * 25_000
+						+ 100_000_000,
```

Comment:
Out of curiosity, where `25_000` and `100_000_000` come from?

Reply:
This is a formula given to us by Moonbeam. Would you like us to add a comment with that info?

Reply:
I think it doesn't does damage 😃, or even `fn moombeam_formula(xcm_domain) -> Weight`. Up to you!